FirebaseUI.podspec

```diff
@@ -1,16 +1,17 @@
 Pod::Spec.new do |s|
   s.name         = 'FirebaseUI'
-  s.version      = '0.4.0'
+  s.version      = '0.5.0'
   s.summary      = 'UI binding libraries for Firebase.'
   s.homepage     = 'https://github.com/firebase/FirebaseUI-iOS'
-  s.license      = { :type => 'Apache License, Version 2.0', :file => 'LICENSE' }
+  s.license      = { :type => 'Apache 2.0' }
   s.author       = 'Firebase'
-  s.source       = { :http => 'https://dl.google.com/firebase/firebaseui/ios/0_4_0/FirebaseUIFrameworks.zip' }
+  s.source       = { :http => 'https://dl.google.com/dl/firebase/firebaseui/ios/0_5_0/FirebaseUIFrameworks.zip' }
```

Comment:
Yeah, it's gonna be released soon :) in the future I hope to just upload zips of the binaries with each github release, so we don't have to rely on this annoying internal google nonsense